data/libs/Vector.lua

```diff
@@ -6,9 +6,9 @@ local Vector = {}
 local function binaryOp(a, b, op)
 	local typ = type(b)
 	if(typ == "number") then
-		return Vector(a.x + b, a.y + b, a.z + b)
```

Comment:
Once the basics are in, I'll try to work in smaller increments (and thus more logical steps) ;)